agent/agent.go

```diff
@@ -1050,6 +1050,7 @@ func (a *Agent) upsertCheckRun(ctx context.Context, tc *taskContext) error {
 	}
 
 	if err = a.comm.UpsertCheckRun(ctx, tc.task, *checkRunOutput); err != nil {
+		tc.logger.Task().Debugf("Error upserting checkRun: '%s'", err.Error())
```

Comment:
Should we maybe do it here: 

```
err = a.upsertCheckRun(ctx, tc)
	if err != nil {
		grip.Error(errors.Wrap(err, "upserting checkrun"))
	}
```

so that it logs a wider range of errors?

Reply:
Also, this should probably be .Error()

Reply:
updated